src/style/styles.ts

```diff
@@ -0,0 +1,58 @@
+const drawerWidth = 240;
+
+export const styles = theme => ({
```

Comment:
@tommysullivan / @bob-bins  How does the styles work ? We define the css here but how it is being used ? I see styles being defined  in other files too.